src/test/kotlin/no/nav/syfo/application/SenOppfolgingServiceSpek.kt

```diff
@@ -194,10 +197,10 @@ class SenOppfolgingServiceSpek : Spek({
 
                 val pKandidat = database.getSenOppfolgingKandidater().first()
                 pKandidat.uuid shouldBeEqualTo kandidat.uuid
-                pKandidat.status shouldBeEqualTo SenOppfolgingStatus.FERDIGBEHANDLET.name
+                pKandidat.status shouldBeEqualTo SenOppfolgingStatus.FERDIGBEHANDLET
                 val pVurdering = database.getSenOppfolgingVurderinger().first()
                 pVurdering.kandidatId shouldBeEqualTo pKandidat.id
-                pVurdering.type shouldBeEqualTo VurderingType.FERDIGBEHANDLET.name
+                pVurdering.type shouldBeEqualTo VurderingType.FERDIGBEHANDLET
```

Comment:
Bør sjekke at verdien for begrunnelse som blir lest fra databasen er den samme som ble skrevet.

Reply:
Bra sett! Legger det til 🤩